tests/qa_features/bootstrap/FeatureContext.php

```diff
@@ -0,0 +1,657 @@
+<?php
+
+use Behat\Behat\Context\ClosuredContextInterface;
+use Behat\Behat\Context\TranslatedContextInterface;
+use Behat\Behat\Context\BehatContext;
+use Behat\Behat\Exception\PendingException;
+use Behat\Gherkin\Node\PyStringNode;
+use Behat\Gherkin\Node\TableNode;
+use Behat\Gherkin\Node\ScenarioNode;
+
+/**
+ * Features context for Behat feature testing
+ */
+class FeatureContext extends BehatContext {
```

Comment:
just noticing how they're different in that they are descriptive of the command being run, vs. our other test suite which is more of "when I run this command, I expect this output". Different kinds of tests though. So yes, merge away!